mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -284,6 +285,35 @@ case class ScalaPresentationCompiler(
           ).selectionRange().asJava
       }
     }
+  end selectionRange
+
+  def expandRangeToEnclosingApply(
```

Comment:
This was needed to get the actual type and not the generic type. I also caused the changes to hover (which are not 100% satifactory, but I think it's still better.)